contracts/payloads/IGP30/PayloadIGP30.sol

```diff
@@ -0,0 +1,915 @@
+        calldatas[0] = abi.encode(ADDRESS_THIS, "execute()", abi.encode());
+
+        uint256 proposedId = GOVERNOR.propose(
+            targets,
+            values,
+            signatures,
+            calldatas,
+            description
+        );
+
+        require(proposedId == PROPOSAL_ID, "PROPOSAL_IS_NOT_SAME");
+    }
+
+    function execute() external {
+        require(address(this) == address(TIMELOCK), "not-valid-caller");
+
+        /// Action 1: Set wBTC token config and market rate curve on liquidity.
+        action1();
+
+       /// Action 2: Deploy wBTC/USDC and wBTC/USDT vaults.
+       action2();
+
+       /// Action 3: Deploy wBTC/ETH and ETH/wBTC vaults.
+       action3();
+
+       /// Action 4: Deploy wstETH/wBTC and weETH/wBTC vaults.
+       action4();
+
+       /// Action 5: Clone from old vault config to new vault
+       action5();
+
+    }
+
+    function verifyProposal() external view {}
+
+    /***********************************|
+    |     Proposal Payload Actions      |
+    |__________________________________*/
+
+    /// @notice Action 1: Set wBTC token config and market rate curve on liquidity.
+    function action1() internal {
+        {
+            AdminModuleStructs.RateDataV1Params[]
+                memory params_ = new AdminModuleStructs.RateDataV1Params[](1);
+
+            params_[0] = AdminModuleStructs.RateDataV1Params({
+                token: wBTC_ADDRESS, // wBTC
+                kink: 80 * 1e2, // 80%
+                rateAtUtilizationZero: 0, // 0%
+                rateAtUtilizationKink: 5 * 1e2, // 5%
+                rateAtUtilizationMax: 100 * 1e2 // 100%
+            });
+
+            LIQUIDITY.updateRateDataV1s(params_);
+        }
+
+        {
+            AdminModuleStructs.TokenConfig[]
+                memory params_ = new AdminModuleStructs.TokenConfig[](1);
+
+            params_[0] = AdminModuleStructs.TokenConfig({
+                token: wBTC_ADDRESS, // wBTC
+                threshold: 0.3 * 1e2, // 0.3
+                fee: 10 * 1e2, // 10%
+                maxUtilization: 100
+            });
+
+            LIQUIDITY.updateTokenConfigs(params_);
+        }
+
+    }
+
+    /// @notice Action 2: Deploy wBTC/USDC and wBTC/USDT vaults.
+    function action2() internal {
+        VaultConfig memory vaultConfig = VaultConfig({
+            // user supply config for the vault on Liquidity Layer.
+            supplyToken: wBTC_ADDRESS,
+            supplyMode: 1, // Mode 1
+            supplyExpandPercent: 25 * 1e2, // 25%
+            supplyExpandDuration: 12 hours, // 12 hours
+            supplyBaseLimitInUSD: 5_000_000, // $5M
+
+            borrowToken: address(0),
+            borrowMode: 1, // Mode 1
+            borrowExpandPercent: 20 * 1e2, // 20%
+            borrowExpandDuration: 12 hours, // 12 hours
+            borrowBaseLimitInUSD: 7_500_000, // $7.5M
+            borrowMaxLimitInUSD: 20_000_000, // $20M
```

Comment:
weETH / WBTC vault is in the only one that has 20M instead of 200M

Reply:
https://instadapp.slack.com/archives/C071TMLTB70/p1719846879284049?thread_ts=1719761810.357469&cid=C071TMLTB70

it should be 20M only, right?

Reply:
Samyak said 200M :smile: 

![image](https://github.com/Instadapp/inst-governance/assets/6112929/62cb17b7-2146-45bd-888e-07f452fd9c54)

Reply:
Ohh! okay, right.